### PR TITLE
update fonts otf -> woff2

### PR DESCRIPTION
**Pull request by derduher**

### CODE REVIEW

File: dist/fonts.min.css [hunk 1]
```diff
@@ -1,2 +1,13 @@
...
+@font-face { font-family: 'SourceSansPro-Regular'; src: url( 'fonts/SourceSansPro-20190717-180900-5e1d162e.woff2' ) format( 'woff2' ); font-weight: normal; font-style: normal; }
```
Comment (m242) [inline dist/fonts.min.css]: Maybe leave the otf file in there for IE11 users even though we aren't technically supporting them?

File: src/fonts.css [hunk 2]
```diff
@@ -1,30 +1,30 @@
 @font-face {
   font-family: 'SourceSansPro-Regular';
-  src: url('fonts/SourceSansPro-Regular.otf') format("opentype");
+  src: url('fonts/SourceSansPro-20190717-180900-5e1d162e.woff2') format('woff2');
```
Comment (m242) [inline src/fonts.css]: Maybe leave the otf file in there for IE11 users even though we aren't technically supporting them?

Comment (derduher) [reply src/fonts.css]: Why though? It'll fallback. Also the app just won't work in IE soon

Comment (m242) [reply src/fonts.css]: Yeah fair, Edge still works.

**Pull request merged**